nemo/collections/tts/modules/attribute_prediction_model.py

```diff
@@ -54,18 +56,18 @@ def __init__(self, in_dim, reduction_factor, norm='weightnorm', non_linearity='r
         reduced_dim = int(in_dim / reduction_factor)
         self.out_dim = reduced_dim
         if self.reduction_factor > 1:
-            fn = ConvNorm(in_dim, reduced_dim, kernel_size=3)
             if norm == 'weightnorm':
-                fn = torch.nn.utils.weight_norm(fn.conv, name='weight')
+                norm_args = {"use_weight_norm": True}
             elif norm == 'instancenorm':
-                fn = nn.Sequential(fn, nn.InstanceNorm1d(reduced_dim, affine=True))
-
+                norm_args = {"norm_fn": MaskedInstanceNorm1d}
```

Comment:
what is the value of `norm_args` when `norm not in ["weightnorm", "instancenorm"]`? it would crash without assigning values to it.

Reply:
Done.